_includes/manuals/2.0/4.2.5_matchers.md

```diff
@@ -1,5 +1,5 @@
 
-The same smart matching technology underpins both smart variables and smart class parameters, so is described below.  It provides the following features for each parameter:
+The same smart matching technology underpins smart class parameters, so is described below.  It provides the following features for each parameter:
```

Comment:
I think the first sentence needs to be changed a bit to make sense now that it only refers to smart class params